core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -21,19 +21,35 @@
  */
 
 oppia.factory('SimpleEditorManagerService', [
-  'explorationInitStateNameService', 'OutcomeObjectFactory',
-  'QuestionListObjectFactory', 'QuestionObjectFactory',
-  'SimpleEditorShimService', 'StatesToQuestionsService',
+  'AnswerGroupObjectFactory', 'explorationInitStateNameService',
+  'OutcomeObjectFactory', 'QuestionObjectFactory', 'QuestionListObjectFactory',
+  'RuleObjectFactory', 'StatesToQuestionsService', 'SimpleEditorShimService',
   function(
-    explorationInitStateNameService, OutcomeObjectFactory,
-    QuestionListObjectFactory, QuestionObjectFactory,
-    SimpleEditorShimService, StatesToQuestionsService) {
+      AnswerGroupObjectFactory, explorationInitStateNameService,
+      OutcomeObjectFactory, QuestionObjectFactory, QuestionListObjectFactory,
+      RuleObjectFactory, StatesToQuestionsService, SimpleEditorShimService) {
     var data = {
       title: null,
       introductionHtml: null,
       questionList: null
     };
 
+    var DEFAULT_INTERACTION_PROPERTIES = {
+      MultipleChoiceInput: {
+        CUSTOMIZATION_ARGS:  {
```

Comment:
Why do we replicate this in DEFAULT_INTERACTION (below) and here? We should probably only have just one copy...